BioArchLinux/vsearch/lilac.yaml

```diff
@@ -0,0 +1,13 @@
+build_prefix: extra-x86_64
+maintainers:
+  - github: althonos
+    email: martin.larralde@embl.de
```

Comment:
If this is an outlook email, our server will fail, so better choose another, you can also have a bioarchlinux email if you are delighted.

Reply:
I changed to my Gmail burner.